app/src/main/res/values/strings.xml

```diff
@@ -216,4 +216,28 @@
     <string name="settings_locale">Language</string>
     <string name="settings_key_locale" translatable="false">pref_locale</string>
     <string name="settings_system_locale">System</string>
+
+    <string name="setting_key_theme_color" translatable="false">pref_theme_color</string>
+    <string name="settings_theme_color">Theme color</string>
+    <string name="settings_app_theme">App theme</string>
```

Comment:
Can we name this one `Catima` please, because it are the "official" colours :)

Reply:
done:)